styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/hub-hero-no-overlay.twig

```diff
@@ -0,0 +1,14 @@
+{#
```

Comment:
Due to the way that PatternLab previews the twig files in tabs, I'd like to start putting the copyright docblock at the bottom of the twig (so that it's not just "Copyright Palantir" that you see in the preview when you open it). I've run this by all the relevant folks at Palantir and they're fine with it. Please move this to the bottom of the file and do the same for all future twig files in this repo.